tests/test_servers/test_servlet.py

```diff
@@ -85,3 +85,61 @@ def test_get_obj_does_not_exist(self, test_servlet):
         assert resp.output_type == "exception"
         error = deserialize_data(resp.data["error"], "pickle")
         assert isinstance(error, KeyError)
+
+    @pytest.mark.level("local")
+    def test_cluster_checks_thread_is_running(self, test_cluster_servlet):
+
+        resp_status_checks_thread = ObjStore.call_actor_method(
```

Comment:
is this obj store method only used for testing purposes? is there a way we can check it's running without it, or alternatively if it's not running would we easily know that in the other tests and having this test is not necessary?

Reply:
Not sure there is an other way to test it, that will not require creating an additional method in the ClusterServlet. 
1. We need to call this thread.is_alive() on the ClusterServlet itself, and we'll be able to do it only if the cluster servlet will have the `_cluster_periodic_thread_alive` method. Another way to test it is to get the `cluster_checks_thread` itself, but this will also require creating a get method.
2. The other tests executes and tests the specific logic of the periodic_cluster_checks thread. We executing the helping functions (`acheck_cluster_logs`, `acheck_cluster_status`) locally. So those tests are not really check that the thread itself is running.